examples/serialization/load.py

```diff
@@ -0,0 +1,28 @@
+import chainer
+import chainer.functions as F
+import chainer.links as L
+
+n_in = 784
+n_units = 100
+n_out = 10
+
+# Network definition
+class MLP(chainer.Chain):
```

Comment:
How about moving the model definition into a separate file?

Reply:
Thanks. moved